src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/RedundantModifierCheck.java

```diff
@@ -61,7 +62,7 @@
  * Interfaces by definition are abstract so the {@code abstract}
  * modifier on the interface is redundant.
  * </p>
- * <p>Classes inside of interfaces by definition are public and static,
+ * <p>Type declarations inside of interfaces by definition are public and static,
```

Comment:
here it is `Type declarations` but above it is mentioned that they are nested. I think it should be mentioned here as well.

Reply:
Done